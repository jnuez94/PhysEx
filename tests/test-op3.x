int a;
int b;
int c;
int d;
int e;
int f;

void func main() {
	a = 12;
	b = 5;
	c = -6;
	d = 52;
	e = 3;
	f = -2;

	printi(a+b);
	printi(a+c+d+f);
	printi(e-f);
	printi(f-c);
	printi(f*e);
	printi(a*b);
	printi(d/f);
	printi(a/c);
	printi(a-d+b+b);
	printi(f+e+b-a);
	printi(b*c/e);
	printi(c/f*b);
	printi(f*e-c);
	printi(f*e+a/c);
}
