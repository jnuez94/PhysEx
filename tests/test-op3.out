17
56
5
4
-6
60
-26
-2
-30
-6
-10
15
0
-8
12
